services/galley/src/Galley/API/Action.hs

```diff
@@ -1111,9 +1112,10 @@ pushTypingIndicatorEvents ::
   Sem r ()
 pushTypingIndicatorEvents qusr tEvent users mcon qcnv ts = do
   let e = Event qcnv Nothing qusr tEvent (EdTyping ts)
-  for_ (newPushLocal ListComplete (qUnqualified qusr) (ConvEvent e) (userRecipient <$> users)) $ \p ->
-    push1 $
-      p
-        & pushConn .~ mcon
-        & pushRoute .~ RouteDirect
-        & pushTransient .~ True
+  for_ (newPushLocal (qUnqualified qusr) (toJSONObject e) (userRecipient <$> users)) $ \p ->
+    pushNotifications
```

Comment:
Should we qualify this everywhere by convention, maybe?
Maybe such a convention can help when reading the code to see right away if it's a local function or a subsystem API call.

Reply:
Its a bit too much to follow conventions like this. I would like it if we could just not have to qualify these. I will see if I can get rid of the collisions in other places.